health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreenParent/SidePanel.js

```
@@ -0,0 +1,243 @@
+                  fill={iconColor}
+                  style={{
+                    transform: isOpen
+                      ? position === "left"
+                        ? "rotate(180deg)"
+                        : "rotate(0deg)"
+                      : position === "left"
+                      ? "rotate(0deg)"
+                      : "rotate(180deg)",
+                    transition: "transform 0.3s ease",
+                  }}
+                ></SVG.ChevronRight>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)